test/hls/hls_parser_unit.js

```diff
@@ -1956,32 +1956,6 @@ describe('HlsParser', () => {
           .toBeRejectedWith(Util.jasmineError(error));
     }
 
-    it('if unable to guess mime type', async () => {
```

Comment:
Can you please add a new test case for the fallback to mp4?  You could use this as a template, but the new test should not be nested within `describe('Errors out', () => {`

Thanks!